.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 npm-debug.log
 .DS_Store
 coverage/
+.vscode
```

Comment:
Missing newline at EOF. [Reasoning](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)